src/DynamoCore/DSEngine/EngineController.cs

```diff
@@ -624,9 +626,9 @@ public CompilationServices(ProtoCore.Core core)
             compilationCore = core;
         }
 
-        public bool PreCompileCodeBlock(ref ParseParam parseParams)
+        public bool PreCompileCodeBlock(ref ParseParam parseParams, ElementResolver elementResolver)
```

Comment:
Please remove this new argument (refer to another comment below about `PreCompileCodeBlock`).